### DISCUSSION TITLE
Azure DevOps Integration (or generic 'work item framework' integration mapping)

### DISCUSSION THREAD

**Discussion by josh-symplr** [apache/devlake#897] [Ideas]
as it stands it would be useful to have more work item integrations.  Gitlab/github are good for code integrations, but there are many other work item frameworks out there (Azure DevOps being one example as an alternative to Jira)

**Comment by keon94** [comment]
Thanks for your suggestion. An initial release of an Azure Devops plugin is targetted for v0.17.0.

**Reply by alexvaccaro** [reply]
Hi @keon94 , I have tried the 17 alpha with the Azure DevOps integration, however it doesn't seem that the Issue tracking Data Model is supported, are there plans to support it at some point?
